IanGRap/Squish
# Options menu

Contributors: Austin

Goal:
- Create a system where players can turn on and off certain visual options
- Options should persist even if you turn the game off

What Changed:
- GameSettings.cs:
    - A static class that reads and writes settings to a json file
- GameSettingsManager.cs
    - Applies settings chosen in the options menu
- UIOptionsMenu.cs
    - Controls the logic of the options menu
- LobbyStateManager.cs
    - Added a new state called OptionsMenu and a new camera location

How to Test:
- Open the lobby scene and make sure you can change options, and see the changes immediately
- Make sure that your options are persistent even if you close the game
- Test in a build
